controller/apierror/messages.go

```diff
@@ -198,4 +198,12 @@ const (
 	ServerTooManyRequestsCode    string = "SERVER_TOO_MANY_REQUESTS"
 	ServerTooManyRequestsMessage string = "Too many requests to alter state have been issued. Please slow your request rate or try again later."
 	ServerTooManyRequestsStatus  int    = http.StatusTooManyRequests
+
+	ServerNotRunningInClusterModeCode    string = "SERVER_NOT_RUNNING_IN_CLUSTER_MODE"
```

Comment:
Cluster mode? HA mode? I have a capability in the edge section called `HA_CONTROLLER`.

I think we should try to be consistent.

Reply:
Makes sense, I'll change it to HA

Reply:
Done